x-pack/test/apm_api_integration/tests/suggestions/suggestions.spec.ts

```diff
@@ -9,12 +9,14 @@ import {
   SERVICE_NAME,
   TRANSACTION_TYPE,
 } from '../../../../plugins/apm/common/elasticsearch_fieldnames';
+import archives_metadata from '../../common/fixtures/es_archiver/archives_metadata';
```

Comment:
it's out of scope for this issue, but I think we should aim to use synthtrace everywhere rather than load the archives. that allows us to move away from snapshot testing to logical assertions.